skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/Sak.kt

```diff
@@ -39,15 +39,15 @@ fun Route.sakRoute(
         route("/bestillBrev") {
             post<LegacyBrevService.BestillDoksysBrevRequest>("/doksys") { request ->
                 val sak = call.attributes[AuthorizeAnsattSakTilgang.sakKey]
-                call.respond(legacyBrevService.bestillOgRedigerDoksysBrev(call, request, enhetsId = sak.enhetId, sak.saksId))
+                call.respond(legacyBrevService.bestillOgRedigerDoksysBrev(call, request, enhetsId = request.enhetsId, sak.saksId))
```

Comment:
Er det noen form for autorisering av at enhetsId er en enhet saksbehandler faktisk har tilgang til?

Reply:
Kjempe :)